tests/conftest.py

```diff
@@ -8,6 +8,41 @@
 from .testdata import testdata
 
 
+class Helpers:
+    """
+    Class containing helper methods for performing tests.
+    """
+
+    @staticmethod
+    def compare_data_frames(df_ref, df):
```

Comment:
Let's consider using it eventually.

Reply:
I think you should rely on https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_frame_equal.html#pandas-testing-assert-frame-equal instead of rolling your own.